aviatrix_ha.py

```diff
@@ -366,7 +366,7 @@ def set_environ(client, lambda_client, controller_instanceobj, context,
         'KEY_NAME': keyname,
         'CTRL_SUBNET': ctrl_subnet,
         'AVIATRIX_TAG': os.environ.get('AVIATRIX_TAG'),
-        'API_PRIVATE_ACCESS': "False",
+        'API_PRIVATE_ACCESS': os.environ.get('API_PRIVATE_ACCESS'),
```

Comment:
This breaks the code for people not using it in private subnet upon initialization. Add a default value of false
Change it to os.environ.get("API_PRIVATE_ACCESS", "False")